misk/web/tabs/web-actions/package.json

```diff
@@ -15,20 +15,20 @@
     "prebuild": "npm run-script lint",
     "reinstall": "rm -rf node_modules && npm run-script install",
     "start": "npm run-script prebuild && cross-env NODE_ENV=development webpack-dev-server",
-    "test": "jest --passWithNoTests",
+    "test": "jest --passWithNoTests --maxWorkers=2",
```

Comment:
what does this do?

Reply:
Let's misk-web tests run in CircleCI! Context: https://github.com/cashapp/misk-web/pull/714